.github/workflows/nightly-perf-tests.yml

```diff
@@ -0,0 +1,145 @@
+name: Nightly Perf Tests
+
+on:
+  schedule:
+    - cron: "0 6 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ github.ref }}-${{ github.sha }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ github.ref }}
+
+      - name: Gradle build service
```

Comment:
I think the nightly perf test should run against the perf environment, which (should) already have a catalog service deployed to it. If so, some of this setup won't be necessary; we don't need to build or run postgres, for example.

Reply:
The catalog config for perf is using the perf server (not localhost):

https://github.com/DataBiosphere/terra-data-catalog/blob/main/integration/src/main/resources/servers/catalog-perf.json#L6

(ignore the use of `policyManagerUri`, this will be fixed in a future PR)